packages/api/src/apps/api/app.ts

```diff
@@ -29,12 +29,12 @@ export default async (env: ProcessEnv) => {
   // we need web3 for syth price feeds
   const web3 = new Web3(env.CUSTOM_NODE_URL);
 
-  // how many blocks to skip before running updates on contract state
-  const updateBlocks = Number(env.UPDATE_BLOCKS || 1);
+  // how often to run expensive state updates, defaults to 10 minutes
+  const updateRateS = Number(env.UPDATE_RATE_S || 10 * 60);
```

Comment:
I believe it was set to 100 blocks in prod, which was roughly 25 minutes. I've shortened that to 10 minutes by default because it seems like we want to get state updates in a bit quicker when things expire.  This is configurable though.